packages/docs/docs/100-getting-started/index.md

```diff
@@ -0,0 +1,24 @@
+---
+---
+
+# Getting started
+
+## Quick start: Node.js
```

Comment:
Why quick start and getting started? I suggest you use the same terminology everywhere, how about keeping it minimal and just

Express.js
Nextjs
React